src/styles/animations.css

```diff
@@ -0,0 +1,29 @@
+@keyframes fade-in {
+	from {
+		opacity: 0;
+	}
+	to {
+		opacity: 1;
+	}
+}
+
+.fade-in {
+	animation-duration: 0.2s;
```

Comment:
Do you think this might be better as a var? ie. `short-animation-duration` ? I believe the material style approach suggests using 0.2 for small transitions and 0.5 for full page transitions.